packages/web/components/cards/rewards-card.tsx

```diff
@@ -7,13 +7,15 @@ export const RewardsCard: React.FC<{
   title: string;
   tooltipContent: string;
   onClick: () => void;
-}> = ({ title, tooltipContent, onClick }) => {
+  bgImage?: JSX.Element;
+}> = ({ title, tooltipContent, onClick, bgImage = null }) => {
   return (
     <div
-      className="flex w-full flex-grow cursor-pointer flex-col rounded-xl border-2 border-osmoverse-600"
+      className="relative flex w-full flex-grow cursor-pointer flex-col overflow-hidden rounded-xl border-2 border-osmoverse-600"
```

Comment:
Should we assume the user of this component wants relative positioning, or should that be passed via classnames from the parent? What is your rationale for adding relative here?